src/admin/WL_Metabox/class-wl-metabox-field.php

```diff
@@ -423,6 +423,29 @@ protected function get_add_button_html( $count ) {
 		return '';
 	}
 
+	/**
```

Comment:
@navdeepsingh besides the fact that you need to change version number and add comments for additional parameters here, I have the feeling that you don't need to create another function very similar, to `get_add_button_html`, you could just add a couple optional parameters to it.

Reply:
You are right. Instead, @ziodave suggested creating a new function instead of editing the current one.